CONTRIBUTING.md

```diff
@@ -26,15 +26,23 @@ If you feel like giving a hand, here are some ways:
     - __Contribution to Optuna includes not only sending pull requests, but also writing down your comments on issues and pull requests by others, and joining conversations/discussions on [Gitter](https://gitter.im/optuna/optuna).__
     - Also, sharing how you enjoy Optuna is a huge contribution! If you write a blog, let us know about it!
 
-If you write code, we have some conventions as follows.
 
-- [Guidelines](#guidelines)
+# Pull Request Guidelines
+
+If you make a pull request, please follow the guidelines below:
+
+- [Setup Optuna](#setup-optuna)
+- [Checking the Format, Coding Style, and Type Hints](#checking-the-format-coding-style-and-type-hints)
+- [Documentation](#documentation)
 - [Unit Tests](#unit-tests)
 - [Continuous Integration and Local Verification](#continuous-integration-and-local-verification)
 - [Creating a Pull Request](#creating-a-pull-request)
-- [Learning Optuna's Implementation](#learning-optunas-implementation)
 
-## Guidelines
+Detailed conventions and policies to write, test, and maintain Optuna code are described in the [Optuna Wiki](https://github.com/optuna/optuna/wiki).
+
+- [Coding Style Conventions](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions)
+- [Deprecation Policy](https://github.com/optuna/optuna/wiki/Deprecation-policy)
+- [Test Policy](https://github.com/optuna/optuna/wiki/Test-Policy)
 
 ### Setup Optuna
```

Comment:
```suggestion
## Setup Optuna
```

Reply:
Thanks for pointing this out. After [this change](https://github.com/optuna/optuna/pull/3726#issuecomment-1169485567), "Setup Optuna" is actually the 3rd level. Thus, I'll leave this line as it is.